demo/x-local-translate.html

```diff
@@ -76,6 +77,10 @@
         }
       },
 
+      _changeLanguage() {
+        this.$.switch.checked = this.language === 'fr';
```

Comment:
Isn't this a bit circular? When you flick the toggle, it calls `_toggle`, which sets `this.language`, which fires this observer?